packages/components/src/components/button/button.tsx

```diff
@@ -61,6 +61,10 @@ export class Button {
   @Prop() download?: string;
   /** (optional) Set `tabindex` in the inner button or link element */
   @Prop() innerTabindex?: number;
+  /** (optional) Button aria-label */
+  @Prop() ariaLabel?: string;
```

Comment:
I think this prop should be renamed to `innerAriaLabel` but I'm not sure. Because `aria-label` can practically be added to anything and `scale-button` is indeed an element, now you would get 2 aria-labels instead of the one you actually want in the actual button inside the shadow DOM. Would you agree?